AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -261,6 +283,22 @@ internal class CardViewController: FormViewController {
 
         return additionalItem
     }()
+    
+    internal lazy var socialSecurityNumberItem: FormTextInputItem = {
+        let securityNumberItem = FormTextInputItem(style: formStyle.textField)
+        securityNumberItem.title = localizedString(.boletoSocialSecurityNumber, localizationParameters)
+        securityNumberItem.placeholder = "123.456.890-12"
```

Comment:
this can be added to the strings file, this way merchants can override this and provide their own placeholder